python/hpsmc/tools.py

```diff
@@ -647,7 +651,7 @@ def required_parameters(self):
         return ['detector', 'steering_files']
     
     def optional_parameters(self):
-        return ['run_number', 'skip_events', 'nevents', 'event_print_interval']
+        return ['run_number', 'skip_events', 'nevents', 'event_print_interval', 'conditions_url']
```

Comment:
This isn't correct. You don't want to include `conditions_url` as a parameter. It is a config setting, which is not the same thing. The parameters are read from the job.json file but the config is not.